freqtrade/rpc/rpc.py

```diff
@@ -157,15 +160,16 @@ def _rpc_status_table(self, stake_currency: str,
                 profit_str = f'{trade_perc:.2f}%'
                 if self._fiat_converter:
                     fiat_profit = self._fiat_converter.convert_amount(
-                            trade_profit,
-                            stake_currency,
-                            fiat_display_currency
-                        )
+                        trade_profit,
+                        stake_currency,
+                        fiat_display_currency
+                    )
                     if fiat_profit and not isnan(fiat_profit):
                         profit_str += f" ({fiat_profit:.2f})"
                 trades_list.append([
                     trade.id,
-                    trade.pair,
+                    trade.pair + ['', '*'][trade.open_order_id is not None
+                                           and trade.close_rate_requested is None],
```

Comment:
```
trade.pair + '*' if (trade.open_order_id is not None
                     and trade.close_rate_requested is None) else ''
```
-- proposing this can easier for the eyes...

Reply:
I actually find the one I have used easier to read, but I can change it if this is a guideline in the codebase.

Reply:
The reason I'm filtering out close rate requested is to avoid showing that asterisk for sell orders, because what the asterisk means in this context is that the position is not yet opened, so you can't consider the Profit. Possibly in a future change this table can be enhanced to provide more complete information including showing sell orders?

Reply:
@xmatthias @hroff-1902 I just had an idea to show double asterisks for sell orders, and keep the single asterisk for buy orders. If you like it, will add it in.

Reply:
maybe some Unicode emojis can be placed, carefully choosen (not in this PR in this case, along with other emojis)?

Reply:
Good idea for the emojis PR. For this PR, double asterisks or a different symbol?

Reply:
whatever (same '*' looks also okay for me)

Reply:
But, as I said the whole point for me was to be able to tell whether you're in the position yet when looking at the status table.

Reply:
but "time" will tell ... i mean - if an order has an * and is open since "2 hours" - then it's probably clear it's a sell-order that's pending.

however, double ** is also fine ... we should just make sure to document this somewhere (maybe the telegram docs? or using `/help` ?)  ... otherwise it'll raise questions ... 

Apart from that (thinking about this further now) - are you sure about this location for adding the *? 
that's a "telegram only" - visual change i would assume - i'd ratter add a `"open_order": "buy"`, "open_order": "sell"` or `"open_order":None` to the output json so it's rest-api friendly .... ? 

Also - the info is missing from `/status` (without table) ...

Reply:
but anyway you can leave that as a "future improvement" - it's just something i noticed while clicking the wrong button :laughing:

Reply:
Yes, I was actually considering a follow-up PR to bring more consistency on /status. The way I thought about it is /status for a verbose view, while /status table is for a more compact one, so the asterisks go in line with this rationale. I'll correct /status table for two asterisks for now, and mention it in the docs.